target/linux/bcm27xx/bcm2710/config-5.10

```diff
@@ -364,7 +363,6 @@ CONFIG_RESET_SIMPLE=y
 CONFIG_RFS_ACCEL=y
 CONFIG_RODATA_FULL_DEFAULT_ENABLED=y
 # CONFIG_RPIVID_MEM is not set
-# CONFIG_RPI_AXIPERF is not set
```

Comment:
This options should be available when you activate `CONFIG_KERNEL_PERF_EVENTS` in the OpenWrt configuration. Removing it here will probably cause problems when `CONFIG_KERNEL_PERF_EVENTS` is set.